ena-submission/src/ena_deposition/config.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from dataclasses import dataclass, field
+
+import yaml
+
+
+@dataclass
+class Config:
+    test: bool
+    organisms: dict[dict[str, str]]
+    backend_url: str
+    keycloak_token_url: str
+    keycloak_client_id: str
+    username: str
+    password: str
+    db_username: str
+    db_password: str
+    db_url: str
+    db_name: str
+    unique_project_suffix: str
+    ena_submission_url: str
+    ena_submission_password: str
+    ena_submission_username: str
+    ena_reports_service_url: str
+    github_url: str
+    slack_hook: str
+    slack_token: str
+    slack_channel_id: str
+    metadata_mapping: dict[str, dict[str, list[str]]]
+    metadata_mapping_mandatory_field_defaults: dict[str, str]
+    ingest_pipeline_submitter: str
+    submit_to_ena_prod: bool = False
+    is_broker: bool = False
+    allowed_submission_hosts: list[str] = field(default_factory=lambda: ["https://backend.pathoplexus.org"])
+    min_between_github_requests: int | None = 2
+    time_between_iterations: int | None = 10
+    min_between_ena_checks: int | None = 5
+    log_level: str = "DEBUG"
+    ena_checklist: str | None = None
+
+
+def secure_ena_connection(config: Config):
+    submit_to_ena_prod = config.submit_to_ena_prod
+    if submit_to_ena_prod and (config.backend_url not in config.allowed_submission_hosts):
+        logging.warn("WARNING: backend_url not in allowed_hosts")
+        submit_to_ena_prod = False
+    submit_to_ena_dev = not submit_to_ena_prod
+
+    if submit_to_ena_dev:
+        config.test = True
+        logging.info("Submitting to ENA dev environment")
+        config.ena_submission_url = "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/submit"
+        config.github_url = "https://raw.githubusercontent.com/pathoplexus/ena-submission/loculus_test/test/approved_ena_submission_list.json"
```

Comment:
```suggestion
        config.github_url = "https://raw.githubusercontent.com/pathoplexus/ena-submission/main/test/approved_ena_submission_list.json"
```
revert before merging